scripts/update-fixtures.sh

```diff
@@ -112,6 +112,16 @@ zip=60304\
 &authority=il-state-of-illinois" \
  | jq . > test/fixtures/il-60304-state-utility-lowincome.json
 
+ curl \
```

Comment:
I didn't realize we have to add to this script as well! I've wrongly assumed the script was aware of the fixtures and updated them accordingly opps. TIL

Reply:
You're right though that this is not how one would reasonably expect it to work! These are basically snapshot tests and should work as such. Unfortunately I don't think tapjs supports that concept very well; yet another reason to switch to Jest?

Reply:
I'm not sure jest does either out of the box either. I've used something like this in the past https://github.com/philschatz/fetch-vcr that you can configure to "record" the response once and leverage it later. I think it has built in functionality to replace the recorded versions. So, it could possibly be worth exploring.